CHANGES.txt

```diff
@@ -1,6 +1,10 @@
 Version history
 ===============
 
+3.1
+
+- changes for Elgg 3.0
+
```

Comment:
Please don't touch changelog as this is done upon release by a script of ours

Reply:
reverted back to original file